src/components/ProfileTree/ProfileBranch.js

```diff
@@ -169,14 +190,18 @@ export const ProfileBranch = memo(({
                 nonMatchData.map((item, i) => (
                   <ProfileBranch
                     key={`profile-branch-${i}`}
-                    entityKey={`${camelCase(item.contentType)}s`}
-                    recordData={item.content}
-                    contentData={item.childSnapshotWrappers}
-                    record={record}
+                    reactTo="NON_MATCH"
```

Comment:
Consider moving this value to a constant.

Reply:
Done